js/main.js

```diff
@@ -0,0 +1,27 @@
+
+//Функция, возвращающая случайное целое число из переданного диапазона включительно
+function returnsRandomNumberFromInterval(min, max) {
+
+  //Проверка числа на >= 0 и корректность диапазона
```

Comment:
```suggestion
  // Проверка числа на >= 0 и корректность диапазона
```

ну ты же не читаешь текст без пробелов, тяжело! Так и тут)